packages/components/src/components/ScalarSidebar/ScalarSidebarGroup.vue

```diff
@@ -45,25 +46,34 @@ defineOptions({ inheritAttrs: false })
 const { cx } = useBindCx()
 </script>
 <template>
-  <li class="contents">
+  <li class="group/item contents">
     <slot
+      :level="level"
       name="button"
       :open="open">
       <ScalarSidebarButton
         is="button"
         :aria-expanded="open"
+        class="text-c-1 bg-b-1"
         :indent="level"
         @click="open = !open">
+        <template #indent>
+          <ScalarSidebarIndent
+            class="mr-0"
+            :indent="level" />
+        </template>
         <template #icon>
-          <ScalarSidebarGroupToggle :open="open" />
+          <ScalarSidebarGroupToggle
+            class="text-c-3"
+            :open="open" />
         </template>
         <slot :open="open" />
       </ScalarSidebarButton>
     </slot>
     <component
       :is="is"
       v-if="open"
-      v-bind="cx('flex flex-col')">
+      v-bind="cx('flex flex-col gap-px')">
```

Comment:
should we keep this .5? cc @cameronrohani 

```suggestion
      v-bind="cx('flex flex-col gap-1/2')">
```

Reply:
We could switch to that if we want, I actually messed up in the client config though and it shouldn't be `1/2` (`1/2` is supposed to be `50%` in tailwind spacing). But we could create `.5px` or `1/2px` as a value.

Reply:
Going to merge for now and we can update later if we want 🫡

Reply:
merge it?